pkg/component/etcd/etcd/bootstrap_test.go

```diff
@@ -271,7 +271,7 @@ var _ = Describe("Etcd", func() {
 				Immutable: ptr.To(true),
 			}
 
-			deploymentWithoutImageVectorOverwriteFor = func(useEtcdWrapper bool) *appsv1.Deployment {
+			deploymentWithoutImageVectorOverwriteFor = func() *appsv1.Deployment {
```

Comment:
This no longer needs to be a func that returns a Deployment. It can be a var of type Deployment.